pkg/ipmi/ipmi.go

```diff
@@ -173,3 +206,43 @@ func (i *IPMI) ShutoffWatchdog() error {
 
 	return nil
 }
+
+func (i *IPMI) AddSystemEventLog(sel *SelEventRecord) error {
```

Comment:
This is adding one Event, right? Not adding an entire log of many events? Based on that, I'd call it `LogSystemEvent`. Go in general prefers using active verbs rather than `add`, `set`, or `get`.

Reply:
I renamed it to `LogSystemEvent`.